internal/gittuf-git/pull/pull.go

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package pull
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/gittuf/gittuf/internal/gittuf-git/args"
+)
+
+func GittufPull(gitArgs args.Args) error {
```

Comment:
Just `Pull` is probably fine.

Is there a reason for this to be a separate package?